packages/eslint-config-datacamp/index.js

```diff
@@ -73,8 +81,27 @@ module.exports = {
     'jest/no-deprecated-functions': 'off', // Needs to know the jest version, not possible from a shared config
     'max-classes-per-file': 'off',
     'no-console': 'error',
+    'no-mixed-requires': 'off', // Deprecated in ESLint 7.0.0, uses node/no-mixed-requires instead
```

Comment:
I am wondering what is the best practice in such case: is it better to do it like here - turn it off, or rather remove it completely from config.

Reply:
@blurbyte The thing is that we added a plugin that replaces these rules. We turned them off so they are not double reported

Reply:
Yeah, probably some presets may have them turned on explicitly (and not updated to eslint 7) so I guess it's better to turn them off.